libgalois/include/katana/PropertyGraph.h

```diff
@@ -293,6 +293,12 @@ class KATANA_EXPORT PropertyGraph {
     return ResultSuccess();
   }
 
+  /// Get the URI for the underlying RDG
+  const katana::Uri& GetRDGUri() { return rdg_.rdg_dir(); }
+
+  /// Set the URI for the underlying RDG
+  void SetRDGUri(const katana::Uri& uri) { return rdg_.set_rdg_dir(uri); }
+
   /// Determine if two PropertyGraphs are Equal
```

Comment:
`SetRDGUri `seems like a version of InformPath that does fewer safety checks (like making sure the provided path is writable).  Also, it doesn't seem right to have both `InformPath` and `SetRDGUri`, do we need both?